documentation/user-interface/welcome-page.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Welcome Page
+permalink: /documentation/user-interface/welcome-page.html
+---
+# Welcome Page
+
+The **Welcome** page introduces you to TestCafe Studio. The page is shown every time when you open the tool. To enable or disable showing the page, use the **Show this page on startup** checkbox. To open the **Welcome** page manually, go to the **Help** menu and select **Welcome**.
+
+![Welcome page](../../images/getting-started/welcome-page.png)
+
+The page allows you to perform the following tasks:
+
+* [Start Recording a Test](#start-recording-a-test)
+* [Open a Test Directory](#open-a-test-directory)
+* [Open Recent Directories](#open-recent-directories)
+* [Open Examples](#open-examples)
+* [Navigate to Documentation and Video Tutorials](#navigate-to-documentation-and-video-tutorials)
+
+## Start Recording a Test
+
+Enter a tested site's URL to the page's input field and click the ![Start recording a test](../../images/guides/record-test-icon.png) **Start recording a test** button to start the recording. TestCafe Studio creates a test, launches a browser with the tested site and starts recording your activity on the webpage. TestCafe Studio adds your actions to the test. You can view the test in the [Test Editor](test-editor.md).
+
+## Open a Test Directory
+
+To open a directory in TestCafe Studio, click the **Open Directory** button and navigate to the directory. TestCafe Studio opens the directory and shows its content in the [Explorer](explorer-panel.md) panel.
+
+## Open Recent Directories
+
+The page lists recently opened test directories. Click a desired directory in the list to open it in TestCafe Studio. You can view the directory's content in the [Explorer](explorer-panel.md) panel.
+
+## Open Examples
+
+Click **Open Examples** to open the test directory with example tests.
```

Comment:
May be add some more words about examples, e.g. smth like "You will see coded test *.js and recorded test". Otherwise the descriptions looks pretty obvious.

Reply:
Agree, we should describe in more details what are these examples and why they can be useful for the user.